gate.py

```
@@ -0,0 +1,348 @@
+
+    return RedirectResponse(f"/gate/complete?token={gate_token}", status_code=303)
+
+
+# ── Completion ────────────────────────────────────────────────────
+
+
+@router.get("/gate/complete", response_class=HTMLResponse)
+async def complete_page(
+    request: Request,
+    token: str = "",
+    pool: asyncpg.Pool = Depends(get_pool),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)